pkg/skuba/actions/node/upgrade/apply.go

```diff
@@ -63,6 +63,8 @@ func Apply(client clientset.Interface, target *deployments.Target) error {
 		return nil
 	}
 
+	// Refreshing cache info about target OS
+	target.IsSUSEOS()
```

Comment:
It might be worth checking the returns of this function: If it's a non SUSE OS or an error, we should indeed bail as soon as possible.

Reply:
Corrected that. Propagated the error and checked for isSUSE bool just in case.